policykit/templates/no-code/main.html

```diff
@@ -607,55 +633,64 @@ <h5 class="modal-title prompt small">Discard the Policy</h5>
     
     }
 
-    const showProcedureVariables = (procedure_value, app) => { 
+    const showProcedureVariables = (procedure_value, app, applied_procedure) => { 
         // if there are no actions selected before, remove the inactive class and empty the section
         if(selected_procedure === null) activate_section("procedure");
         
         // search for data about this selected procedure 
-        selected_procedure = {...procedures[app].find(item => item.value == procedure_value)};;
+        selected_procedure = deepcopy(procedures[app].find(item => item.value == procedure_value));
         update_data_from_procedures();
 
         let procedure_div = add_section_header("procedure", selected_procedure);
         $("#design_procedure-formview").empty();
         $("#design_procedure-formview").append(procedure_div);
         selected_procedure["variables"].forEach((variable, index) => {
+            let set_variable = applied_procedure.variables ? applied_procedure.variables.find(item => item.name == variable.name) : null;
+            if(set_variable) variable.value = set_variable.value;
             addVariableInputBox(variable=variable, id=variable.name, parentDiv=procedure_div, options=entities, datalist=global_datalist, type="procedure")
         })
 
         
     }
         
-    const showTransformerVariables = (transformer_value, app) => {
+    const showTransformerVariables = (transformer_value, app, applied_transformer={}) => {
+        console.log('show transformer variables', applied_transformer)
```

Comment:
same question

Reply:
removed such logs

Reply:
sorry did you add a commit that removed them? I still see them all in the PR.